super.c

```diff
@@ -1044,6 +1045,13 @@ static int apfs_check_features(struct super_block *sb)
 		return -EINVAL;
 	}
 
+	features = le64_to_cpu(vsb_raw->apfs_fs_flags);
+	if (!(features & APFS_FS_UNENCRYPTED)){
+		apfs_warn(sb, "encrypted volumes are not supported");
+		return -EINVAL;
+	}
+
+
```

Comment:
Did you intentionally add two empty lines here? The pattern in the rest of the code seems to be one empty line between code blocks.